models/tasks/activity.go

```diff
@@ -0,0 +1,23 @@
+package tasks // import github.com/mlimaloureiro/golog/models/tasks
+
+import "time"
+
+// TaskActivity represents a date interval of a task execution
+type TaskActivity struct {
+	StartDate time.Time
+	EndDate   time.Time // EndDate.IsZero() indicates a running task, meaning the 0001-01-01 EndDate cannot be represented
```

Comment:
should we have in line comments, or comments in the previous line ? let's also add this to the standards if we don't have it

Reply:
If we have to pick only one i would vote for comments in the previous line. Helps keeping line length low and provides a explanation / comment before the actual code

But do we really need to pick only one? Sometimes i think it makes sense inline, like in this occurrence where it seems to make more sense to have the comment explaining the meaning of `EndDate.IsZero()` after `EndDate` has been declared.